client/js/components/user/DpOrganisationList/DpOrganisationListItem.vue

```diff
@@ -130,11 +130,11 @@ export default {
   },
 
   computed: {
-    ...mapState('orga', {
+    ...mapState('Orga', {
```

Comment:
When you edit an orga and save it, an error is thrown
```
Cannot read properties of undefined (reading 'then')
    at VueComponent.saveOrganisationAction (DpOrganisationListItem.vue?./node_modules/vue-loader/lib/index.js??vue-loader-options!./node_modules/source-map-loader/dist/cjs.js:138:9)
```

Reply:
There are still several errors when editing an orga and saving (and saving does not work).

Reply:
should now be fixed with 0417d796a2a3fe7acae72a688b95608a3194637f